backend/core/src/user/user.dto.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiHideProperty, OmitType } from "@nestjs/swagger"
+import { User } from "../entity/user.entity"
+import { Exclude, Expose } from "class-transformer"
+import { IsString } from "class-validator"
+
+export class UserDto extends OmitType(User, ["passwordHash", "applications", "isAdmin"] as const) {
+  @Exclude()
+  @ApiHideProperty()
+  passwordHash
+
+  @Exclude()
+  @ApiHideProperty()
+  isAdmin
+}
+
+export class UserDtoWithAccessToken extends UserDto {
+  @Expose()
+  accessToken: string
+}
+
+export class UserCreateDto extends OmitType(UserDto, ["id", "createdAt", "updatedAt"] as const) {
+  @Exclude()
+  @ApiHideProperty()
+  id
+
+  @Exclude()
+  @ApiHideProperty()
+  createdAt
+
+  @Exclude()
+  @ApiHideProperty()
+  updatedAt
+
+  @Expose()
+  @IsString()
+  password: string
```

Comment:
Same here with expose.

Reply:
It's UserCreate DTO it must have a password prop